Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
+
+public class Perfection {
+
+    private static Perfection perf;
```

Comment:
Please reconsider this and the constructor. As the methods are static, we don't need to create an object for this class. Just call like this: Perfection.isPerfect(10);